guides/common/modules/proc_configuring-server-to-synchronize-content-over-a-network.adoc

```diff
@@ -45,8 +52,8 @@ This shows some related info under the "CDN Configuration" section in the output
 [options="nowrap" subs="+quotes"]
 ----
 # hammer organization configure-cdn --name="_My_Downstream_Organization_" \
---type=upstream_server/network_sync  # TODO \
---url https://_upstream-server.example.com_ \ // TODO better example URL?
+--type=network_sync \
+--url https://_upstream-server.example.com_ \
 --username _upstream_username_ --password _upstream_password_ \
 --ssl-ca-credential-id _8_ \
```

Comment:
@parthaa I need an output sample, so that I can explain why they need to do it. They need the SSL credential for a later step and I need to tell the user that they get it from the output here.

Reply:
Please, provide